### PR TITLE
button ui

### PR DESCRIPTION
**Pull request by ekiuse**

### CODE REVIEW

**Review by ekiuse — COMMENTED** [review]
nema: update svgIcon

File: src/ui/button/button.module.css [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+    fill: inherit;
     user-select: none;
     cursor: pointer;
     box-sizing: border-box;
 }
+.button.iconButton {
+    width: 4rem;
+    padding: 0;
+}
+a:link {
```
Comment (nexnema) [inline src/ui/button/button.module.css:24]: check this with me

File: src/ui/button/button.module.css [hunk 2]
```diff
@@ -24,10 +37,22 @@
...
     background: var(--onBackgroundColor);
     color: var(--onPrimaryColor);
+    fill: var(--onPrimaryColor);
+}
+.button.tonal {
+    background: var(--onHeaderColor);
+    color: var(--onSurfaceColor);
+    fill: var(--onSurfaceColor);
+}
+.elevated {
```
Comment (nexnema) [inline src/ui/button/button.module.css:53]: add .button

**Review by nexnema — APPROVED** [review]
Approved but check comments

**Pull request merged**